### PR TITLE
Add @SteveEmfore models

### PR DESCRIPTION
**Pull request by duncanspumpkin**
A selection of trains. I've adjusted them to have OpenLoco source game.
One item to discuss all of these have (OGFX) in their strings. Should we remove that? I think yes.

### CODE REVIEW

**Comment by LeftofZen** [comment]
> One item to discuss all of these have (OGFX) in their strings. Should we remove that? I think yes.

Agreed, remove. I would imagine these objects also have the bugged design year?

**Comment by duncanspumpkin** [comment]
> > One item to discuss all of these have (OGFX) in their strings. Should we remove that? I think yes.
> 
> Agreed, remove. I would imagine these objects also have the bugged design year?

I patched the editor last night and used that when editing to address that issue

File: objects/Vehicle/Train/JINTY/.gitkeep
Comment (LeftofZen) [inline objects/Vehicle/Train/JINTY/.gitkeep:1]: I don't see any files added into this folder?

Comment (duncanspumpkin) [reply objects/Vehicle/Train/JINTY/.gitkeep:1]: Argh did i forget to add it.

File: objects/Vehicle/Train/ST8FT/OGST8F.blend
Comment (LeftofZen) [inline objects/Vehicle/Train/ST8FT/OGST8F.blend:1]: missing dat file for this object

File: objects/Vehicle/Train/JINTY/.gitkeep
Comment (LeftofZen) [inline objects/Vehicle/Train/JINTY/.gitkeep:1]: I don't see any files added into this folder?

**Review by LeftofZen — CHANGES_REQUESTED** [review]
ST8FT - missing dat file
JINTY - missing dat file and blend file

File: objects/Vehicle/Train/ST8FT/OGST8F.blend
Comment (duncanspumpkin) [reply objects/Vehicle/Train/ST8FT/OGST8F.blend:1]: fixed

File: objects/Vehicle/Train/JINTY/.gitkeep
Comment (duncanspumpkin) [reply objects/Vehicle/Train/JINTY/.gitkeep:1]: fixed

**Review by LeftofZen — APPROVED** [review]
approving, on the condition the filename extensions are normalised to `.dat` (there are 2 with `.DAT`)

**Pull request merged**